lnwallet/btcwallet/btcwallet.go

```diff
@@ -554,10 +555,26 @@ func unminedTransactionsToDetail(
 		return nil, err
 	}
 
+	// Iterate over all the transaction outputs and  extract the Public Key of the
```

Comment:
Nit: extra space before "extract" and line too long (max 80 chars).

Reply:
fixed